main-project/src/chats/chats-controller.service.ts

```diff
@@ -115,89 +135,70 @@ export class ChatsControllerService {
   }
 
   async inviteUser(
+    manager: EntityManager,
     inviteUser: InviteUserDTO,
```

Comment:
inviteUser를 아래 코드에서 자체적으로 쓰는 곳은 없는 것 같은데
그럼 매개변수 자체에서 펼쳐주시는게 줗을 것 같아요!
```suggestion
 { userNo, targetUserNo }: InviteUserDTO
```